src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Model/V2.Generated.cs

```diff
@@ -0,0 +1,577 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+namespace Microsoft.Health.Dicom.SqlServer.Features.Schema.Model
```

Comment:
where is the SQL file change that generated this? Should'nt this be v3?

Reply:
I switched this to just have V3 be latest!

Reply:
-- I just realized this is just QIDO so V3 can't be latest. Reverting that now! Will change it in the STOW PR.